### ISSUE TITLE
some pods that are stuck in "pending" state after deployment with Kubernetes

### ISSUE THREAD

**Issue by antonadamov** [reportportal/kubernetes#423]
could you please help with an issue I'm facing after deploying RP using the Helm chart? After running [Installing the Chart](https://reportportal.io/docs/installation-steps/DeployWithKubernetes/#installing-the-chart)
commands, multiple pods are stuck in "pending" state. There are no resource issues on the EKS cluster. pods that are stuck:
uat, analyzer, api, jobs, metrics, and opensearch cluster master

**Comment by Pink-Bumblebee** [comment]
@antonadamov , is your problem still actual? 
If yes, please provide additional information about your environment and please attach some logs of services that stuck in "pending" state.

**Comment by c0demonky** [comment]
**[2024-09-16 15:06:01] Waiting for job mgp-tip-report-portal-reportportal-migrations...
[2024-09-16 15:06:07] Waiting for job mgp-tip-report-portal-reportportal-migrations...
[2024-09-16 15:06:13] Waiting for job mgp-tip-report-portal-reportportal-migrations...
[2024-09-16 15:06:19] Waiting for job mgp-tip-report-portal-reportportal-migrations...**

these logs are from the "migrations-waiting-init" pod in one of the pending containers. all pending containers have this pod repeating the same message

**wait-for-it.sh: waiting for mgp-tip-report-portal-postgresql.default.svc.cluster.local:5432 without a timeout
wait-for-it.sh: waiting for opensearch-cluster-master.default.svc.cluster.local:9200 without a timeout
wait-for-it.sh: mgp-tip-report-portal-postgresql.default.svc.cluster.local:5432 is available after 15 seconds
no change**

these are the only logs from the migrations container

**Comment by Pink-Bumblebee** [comment]
@antonadamov, please provide more information: are you trying to install with external Postgres? Have you installed Ingress Controller https://reportportal.io/docs/installation-steps-advanced/ComponentsOverview#gateway ?

**Comment by antonadamov** [comment]
@Pink-Bumblebee , @c0demonky worked on the deployment. 
@c0demonky please provide requested details

**Comment by c0demonky** [comment]
@Pink-Bumblebee we are running this service on an enterprise eks node. I created a dns record and cname to route requests to the services, which are reachable but showing 503 -- aside from UI and Index.
I installed by cloning the master kubernetes repo https://github.com/reportportal/kubernetes/blob/master/reportportal/Chart.yaml and installing helm chart through kubectl

**Comment by hlebkanonik** [comment]
Hi! Regarding our workflow, the Helm Chart first deploys PostgreSQL, RabbitMQ, and OpenSearch. Next, the Migration service prepares the database and OpenSearch. Once the migration is complete, the remaining services will start running.


Could you please share the status and logs of all services? This will help me understand in which step we have a problem.

**Comment by c0demonky** [comment]
**Migrations**
```
wait-for-it.sh: waiting for mgp-tip-report-portal-postgresql.default.svc.cluster.local:5432 without a timeout
wait-for-it.sh: waiting for opensearch-cluster-master.default.svc.cluster.local:9200 without a timeout
wait-for-it.sh: mgp-tip-report-portal-postgresql.default.svc.cluster.local:5432 is available after 15 seconds
no change
```

**Analyzer**, **Analyzer Train**, **Metrics Gatherer**, **OpenSearch**, **Jobs**, **UAT**, **API**
[Uploading reportportal-ui.txt…]()

Defaulted container "analyzer" out of: analyzer, migrations-waiting-init (init)
Error from server (BadRequest): container "analyzer" in pod "mgp-tip-report-portal-reportportal-analyzer-0" is waiting to start: PodInitializing

**Analyzer Train**
Same as above

**Postgres**
```
←[38;5;6mpostgresql ←[38;5;5m17:56:43.75 ←[0m←[38;5;2mINFO ←[0m ==>
←[38;5;6mpostgresql ←[38;5;5m17:56:43.76 ←[0m←[38;5;2mINFO ←[0m ==> ←[1mWelcome to the Bitnami postgresql container←[0m
←[38;5;6mpostgresql ←[38;5;5m17:56:43.85 ←[0m←[38;5;2mINFO ←[0m ==> Subscribe to project updates by watching ←[1mhttps://github.com/bitnami/containers←[0m
←[38;5;6mpostgresql ←[38;5;5m17:56:43.95 ←[0m←[38;5;2mINFO ←[0m ==> Submit issues and feature requests at ←[1mhttps://github.com/bitnami/containers/issues←[0m
←[38;5;6mpostgresql ←[38;5;5m17:56:43.95 ←[0m←[38;5;2mINFO ←[0m ==> Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, includi
ng Software Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit ←[1mhttps://bitnami.com/enterprise←[0m
←[38;5;6mpostgresql ←[38;5;5m17:56:44.05 ←[0m←[38;5;2mINFO ←[0m ==>
←[38;5;6mpostgresql ←[38;5;5m17:56:44.55 ←[0m←[38;5;2mINFO ←[0m ==> ** Starting PostgreSQL setup **
←[38;5;6mpostgresql ←[38;5;5m17:56:44.76 ←[0m←[38;5;2mINFO ←[0m ==> Validating settings in POSTGRESQL_* env vars..
←[38;5;6mpostgresql ←[38;5;5m17:56:44.95 ←[0m←[38;5;2mINFO ←[0m ==> Loading custom pre-init scripts...
←[38;5;6mpostgresql ←[38;5;5m17:56:45.05 ←[0m←[38;5;2mINFO ←[0m ==> Initializing PostgreSQL database...
←[38;5;6mpostgresql ←[38;5;5m17:56:45.45 ←[0m←[38;5;2mINFO ←[0m ==> pg_hba.conf file not detected. Generating it...
←[38;5;6mpostgresql ←[38;5;5m17:56:45.46 ←[0m←[38;5;2mINFO ←[0m ==> Generating local authentication configuration
←[38;5;6mpostgresql ←[38;5;5m17:56:45.86 ←[0m←[38;5;2mINFO ←[0m ==> Deploying PostgreSQL with persisted data...
←[38;5;6mpostgresql ←[38;5;5m17:56:46.25 ←[0m←[38;5;2mINFO ←[0m ==> Configuring replication parameters
←[38;5;6mpostgresql ←[38;5;5m17:56:46.86 ←[0m←[38;5;2mINFO ←[0m ==> Configuring fsync
←[38;5;6mpostgresql ←[38;5;5m17:56:47.05 ←[0m←[38;5;2mINFO ←[0m ==> Configuring synchronous_replication
←[38;5;6mpostgresql ←[38;5;5m17:56:47.85 ←[0m←[38;5;2mINFO ←[0m ==> Loading custom scripts...
←[38;5;6mpostgresql ←[38;5;5m17:56:47.95 ←[0m←[38;5;2mINFO ←[0m ==> Enabling remote connections
←[38;5;6mpostgresql ←[38;5;5m17:56:48.25 ←[0m←[38;5;2mINFO ←[0m ==> ** PostgreSQL setup finished! **

←[38;5;6mpostgresql ←[38;5;5m17:56:48.45 ←[0m←[38;5;2mINFO ←[0m ==> ** Starting PostgreSQL **
2024-09-13 17:56:48.664 GMT [1] LOG:  pgaudit extension initialized
2024-09-13 17:56:48.758 GMT [1] LOG:  starting PostgreSQL 16.4 on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
2024-09-13 17:56:48.758 GMT [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2024-09-13 17:56:48.758 GMT [1] LOG:  listening on IPv6 address "::", port 5432
2024-09-13 17:56:48.764 GMT [1] LOG:  listening on Unix socket "/tmp/.s.PGSQL.5432"
2024-09-13 17:56:48.773 GMT [93] LOG:  database system was shut down at 2024-09-13 17:55:29 GMT
2024-09-13 17:56:48.857 GMT [1] LOG:  database system is ready to accept connections
2024-09-13 18:01:48.873 GMT [91] LOG:  checkpoint starting: time
2024-09-13 18:01:48.886 GMT [91] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.004 s, sync=0.002 s, total=0.014 s; sync files=2, longest=0.001 s, average=0.001 s; distance=0
 kB, estimate=0 kB; lsn=0/1D32BF0, redo lsn=0/1D32BB8
```

**RabbitMQ**
```
←[38;5;6mrabbitmq ←[38;5;5m17:56:51.50 ←[0m←[38;5;2mINFO ←[0m ==>
←[38;5;6mrabbitmq ←[38;5;5m17:56:51.55 ←[0m←[38;5;2mINFO ←[0m ==> ←[1mWelcome to the Bitnami rabbitmq container←[0m
←[38;5;6mrabbitmq ←[38;5;5m17:56:51.56 ←[0m←[38;5;2mINFO ←[0m ==> Subscribe to project updates by watching ←[1mhttps://github.com/bitnami/containers←[0m
←[38;5;6mrabbitmq ←[38;5;5m17:56:51.57 ←[0m←[38;5;2mINFO ←[0m ==> Submit issues and feature requests at ←[1mhttps://github.com/bitnami/containers/issues←[0m
←[38;5;6mrabbitmq ←[38;5;5m17:56:51.57 ←[0m←[38;5;2mINFO ←[0m ==> Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, including
 Software Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit ←[1mhttps://bitnami.com/enterprise←[0m
←[38;5;6mrabbitmq ←[38;5;5m17:56:51.58 ←[0m←[38;5;2mINFO ←[0m ==>
←[38;5;6mrabbitmq ←[38;5;5m17:56:51.66 ←[0m←[38;5;2mINFO ←[0m ==> ** Starting RabbitMQ setup **
←[38;5;6mrabbitmq ←[38;5;5m17:56:51.76 ←[0m←[38;5;2mINFO ←[0m ==> Validating settings in RABBITMQ_* env vars..
←[38;5;6mrabbitmq ←[38;5;5m17:56:51.88 ←[0m←[38;5;2mINFO ←[0m ==> Initializing RabbitMQ...
←[38;5;6mrabbitmq ←[38;5;5m17:56:52.27 ←[0m←[38;5;2mINFO ←[0m ==> Persisted data detected. Restoring...
←[38;5;6mrabbitmq ←[38;5;5m17:56:52.28 ←[0m←[38;5;2mINFO ←[0m ==> No custom scripts in /docker-entrypoint-initdb.d
←[38;5;6mrabbitmq ←[38;5;5m17:56:52.29 ←[0m←[38;5;2mINFO ←[0m ==> ** RabbitMQ setup finished! **

←[38;5;6mrabbitmq ←[38;5;5m17:56:52.38 ←[0m←[38;5;2mINFO ←[0m ==> ** Starting RabbitMQ **
=INFO REPORT==== 13-Sep-2024::17:56:54.367324 ===
    alarm_handler: {set,{system_memory_high_watermark,[]}}
2024-09-13 17:57:06.164211+00:00 [notice] <0.44.0> Application syslog exited with reason: stopped
2024-09-13 17:57:06.170352+00:00 [notice] <0.254.0> Logging: switching to configured handler(s); following messages may not be visible in this log output
2024-09-13 17:57:06.171386+00:00 [notice] <0.254.0> Logging: configured log handlers are now ACTIVE
2024-09-13 17:57:06.180475+00:00 [info] <0.254.0> ra: starting system quorum_queues
2024-09-13 17:57:06.180552+00:00 [info] <0.254.0> starting Ra system: quorum_queues in directory: /opt/bitnami/rabbitmq/.rabbitmq/mnesia/rabbit@mgp-tip-report-portal-rabbitmq-0.mgp-tip-report-portal-rabbitmq-headless.default.svc.c
luster.local/quorum/rabbit@mgp-tip-report-portal-rabbitmq-0.mgp-tip-report-portal-rabbitmq-headless.default.svc.cluster.local
2024-09-13 17:57:06.465144+00:00 [info] <0.268.0> ra system 'quorum_queues' running pre init for 0 registered servers
2024-09-13 17:57:06.476427+00:00 [info] <0.269.0> ra: meta data store initialised for system quorum_queues. 0 record(s) recovered
2024-09-13 17:57:06.563646+00:00 [notice] <0.274.0> WAL: ra_log_wal init, open tbls: ra_log_open_mem_tables, closed tbls: ra_log_closed_mem_tables
2024-09-13 17:57:06.581773+00:00 [info] <0.254.0> ra: starting system coordination
2024-09-13 17:57:06.581860+00:00 [info] <0.254.0> starting Ra system: coordination in directory: /opt/bitnami/rabbitmq/.rabbitmq/mnesia/rabbit@mgp-tip-report-portal-rabbitmq-0.mgp-tip-report-portal-rabbitmq-headless.default.svc.cl
uster.local/coordination/rabbit@mgp-tip-report-portal-rabbitmq-0.mgp-tip-report-portal-rabbitmq-headless.default.svc.cluster.local
2024-09-13 17:57:06.583698+00:00 [info] <0.282.0> ra system 'coordination' running pre init for 1 registered servers
2024-09-13 17:57:06.668317+00:00 [info] <0.283.0> ra: meta data store initialised for system coordination. 1 record(s) recovered
2024-09-13 17:57:06.668546+00:00 [notice] <0.288.0> WAL: ra_coordination_log_wal init, open tbls: ra_coordination_log_open_mem_tables, closed tbls: ra_coordination_log_closed_mem_tables
2024-09-13 17:57:06.676302+00:00 [info] <0.254.0> ra: starting system coordination
2024-09-13 17:57:06.676380+00:00 [info] <0.254.0> starting Ra system: coordination in directory: /opt/bitnami/rabbitmq/.rabbitmq/mnesia/rabbit@mgp-tip-report-portal-rabbitmq-0.mgp-tip-report-portal-rabbitmq-headless.default.svc.cl
uster.local/coordination/rabbit@mgp-tip-report-portal-rabbitmq-0.mgp-tip-report-portal-rabbitmq-headless.default.svc.cluster.local
2024-09-13 17:57:06.977355+00:00 [info] <0.254.0> Waiting for Khepri leader for 30000 ms, 9 retries left
2024-09-13 17:57:07.075467+00:00 [info] <0.254.0> Khepri leader elected
2024-09-13 17:57:07.075550+00:00 [info] <0.254.0> Waiting for Khepri projections for 30000 ms, 9 retries left
2024-09-13 17:57:07.447708+00:00 [notice] <0.293.0> RabbitMQ metadata store: candidate -> leader in term: 5 machine version: 1
2024-09-13 17:57:08.760302+00:00 [info] <0.254.0>
2024-09-13 17:57:08.760302+00:00 [info] <0.254.0>  Starting RabbitMQ 3.13.7 on Erlang 26.2.5 [jit]
2024-09-13 17:57:08.760302+00:00 [info] <0.254.0>  Copyright (c) 2007-2024 Broadcom Inc and/or its subsidiaries
2024-09-13 17:57:08.760302+00:00 [info] <0.254.0>  Licensed under the MPL 2.0. Website: https://rabbitmq.com

  ##  ##      RabbitMQ 3.13.7
  ##  ##
  ##########  Copyright (c) 2007-2024 Broadcom Inc and/or its subsidiaries
  ######  ##
  ##########  Licensed under the MPL 2.0. Website: https://rabbitmq.com

  Erlang:      26.2.5 [jit]
  TLS Library: OpenSSL - OpenSSL 3.0.13 30 Jan 2024
  Release series support status: see https://www.rabbitmq.com/release-information

  Doc guides:  https://www.rabbitmq.com/docs
  Support:     https://www.rabbitmq.com/docs/contact
  Tutorials:   https://www.rabbitmq.com/tutorials
  Monitoring:  https://www.rabbitmq.com/docs/monitoring
  Upgrading:   https://www.rabbitmq.com/docs/upgrade

  Logs: <stdout>

  Config file(s): /opt/bitnami/rabbitmq/etc/rabbitmq/rabbitmq.conf

  Starting broker...2024-09-13 17:57:08.762099+00:00 [info] <0.254.0>
2024-09-13 17:57:08.762099+00:00 [info] <0.254.0>  node           : rabbit@mgp-tip-report-portal-rabbitmq-0.mgp-tip-report-portal-rabbitmq-headless.default.svc.cluster.local
2024-09-13 17:57:08.762099+00:00 [info] <0.254.0>  home dir       : /opt/bitnami/rabbitmq/.rabbitmq
2024-09-13 17:57:08.762099+00:00 [info] <0.254.0>  config file(s) : /opt/bitnami/rabbitmq/etc/rabbitmq/rabbitmq.conf
2024-09-13 17:57:08.762099+00:00 [info] <0.254.0>  cookie hash    : FsaJCHx/tWzBkOWPZ6QKig==
2024-09-13 17:57:08.762099+00:00 [info] <0.254.0>  log(s)         : <stdout>
2024-09-13 17:57:08.762099+00:00 [info] <0.254.0>  data dir       : /opt/bitnami/rabbitmq/.rabbitmq/mnesia/rabbit@mgp-tip-report-portal-rabbitmq-0.mgp-tip-report-portal-rabbitmq-headless.default.svc.cluster.local
2024-09-13 17:57:11.161948+00:00 [info] <0.254.0> Running boot step pre_boot defined by app rabbit
2024-09-13 17:57:11.162030+00:00 [info] <0.254.0> Running boot step rabbit_global_counters defined by app rabbit
2024-09-13 17:57:11.162296+00:00 [info] <0.254.0> Running boot step rabbit_osiris_metrics defined by app rabbit
2024-09-13 17:57:11.162405+00:00 [info] <0.254.0> Running boot step rabbit_core_metrics defined by app rabbit
2024-09-13 17:57:11.163630+00:00 [info] <0.254.0> Running boot step rabbit_alarm defined by app rabbit
2024-09-13 17:57:11.258939+00:00 [info] <0.390.0> Memory high watermark set to 25385 MiB (26618719436 bytes) of 63463 MiB (66546798592 bytes) total
2024-09-13 17:57:11.272563+00:00 [info] <0.392.0> Enabling free disk space monitoring (disk free space: 8266059776, total memory: 66546798592)
2024-09-13 17:57:11.272668+00:00 [info] <0.392.0> Disk free limit set to 50MB
2024-09-13 17:57:11.279103+00:00 [info] <0.254.0> Running boot step code_server_cache defined by app rabbit
2024-09-13 17:57:11.279253+00:00 [info] <0.254.0> Running boot step file_handle_cache defined by app rabbit
2024-09-13 17:57:11.288650+00:00 [info] <0.395.0> Limiting to approx 1048479 file handles (943629 sockets)
2024-09-13 17:57:11.288947+00:00 [info] <0.396.0> FHC read buffering: OFF
2024-09-13 17:57:11.289021+00:00 [info] <0.396.0> FHC write buffering: ON
2024-09-13 17:57:11.289541+00:00 [info] <0.254.0> Running boot step worker_pool defined by app rabbit
2024-09-13 17:57:11.289652+00:00 [info] <0.336.0> Will use 16 processes for default worker pool
2024-09-13 17:57:11.289708+00:00 [info] <0.336.0> Starting worker pool 'worker_pool' with 16 processes in it
2024-09-13 17:57:11.290761+00:00 [info] <0.254.0> Running boot step database defined by app rabbit
2024-09-13 17:57:11.358557+00:00 [info] <0.254.0> Peer discovery: configured backend: rabbit_peer_discovery_k8s
2024-09-13 17:57:11.358778+00:00 [info] <0.254.0> Peer discovery: will register with peer discovery backend rabbit_peer_discovery_k8s
2024-09-13 17:57:12.559714+00:00 [info] <0.254.0> Waiting for Mnesia tables for 30000 ms, 9 retries left
2024-09-13 17:57:12.562210+00:00 [info] <0.254.0> Successfully synced tables from a peer
2024-09-13 17:57:12.571535+00:00 [info] <0.254.0> Waiting for Mnesia tables for 30000 ms, 9 retries left
2024-09-13 17:57:12.571714+00:00 [info] <0.254.0> Successfully synced tables from a peer
2024-09-13 17:57:12.571917+00:00 [info] <0.254.0> Running boot step tracking_metadata_store defined by app rabbit
2024-09-13 17:57:12.572009+00:00 [info] <0.428.0> Setting up a table for connection tracking on this node: tracked_connection
2024-09-13 17:57:12.572085+00:00 [info] <0.428.0> Setting up a table for per-vhost connection counting on this node: tracked_connection_per_vhost
2024-09-13 17:57:12.572207+00:00 [info] <0.428.0> Setting up a table for per-user connection counting on this node: tracked_connection_per_user
2024-09-13 17:57:12.572299+00:00 [info] <0.428.0> Setting up a table for channel tracking on this node: tracked_channel
2024-09-13 17:57:12.572412+00:00 [info] <0.428.0> Setting up a table for channel tracking on this node: tracked_channel_per_user
2024-09-13 17:57:12.572548+00:00 [info] <0.254.0> Running boot step networking_metadata_store defined by app rabbit
2024-09-13 17:57:12.572673+00:00 [info] <0.254.0> Running boot step feature_flags defined by app rabbit
2024-09-13 17:57:12.572879+00:00 [info] <0.254.0> Running boot step codec_correctness_check defined by app rabbit
2024-09-13 17:57:12.572931+00:00 [info] <0.254.0> Running boot step rabbit_exchange_type_consistent_hash_metadata_store defined by app rabbitmq_consistent_hash_exchange
2024-09-13 17:57:12.589594+00:00 [info] <0.254.0> Waiting for Mnesia tables for 30000 ms, 0 retries left
2024-09-13 17:57:12.589739+00:00 [info] <0.254.0> Successfully synced tables from a peer
2024-09-13 17:57:12.589965+00:00 [info] <0.254.0> Running boot step external_infrastructure defined by app rabbit
2024-09-13 17:57:12.590024+00:00 [info] <0.254.0> Running boot step rabbit_event defined by app rabbit
2024-09-13 17:57:12.590277+00:00 [info] <0.254.0> Running boot step rabbit_registry defined by app rabbit
2024-09-13 17:57:12.590413+00:00 [info] <0.254.0> Running boot step rabbit_auth_mechanism_amqplain defined by app rabbit
2024-09-13 17:57:12.590519+00:00 [info] <0.254.0> Running boot step rabbit_auth_mechanism_cr_demo defined by app rabbit
2024-09-13 17:57:12.590606+00:00 [info] <0.254.0> Running boot step rabbit_auth_mechanism_plain defined by app rabbit
2024-09-13 17:57:12.590771+00:00 [info] <0.254.0> Running boot step rabbit_exchange_type_direct defined by app rabbit
2024-09-13 17:57:12.590915+00:00 [info] <0.254.0> Running boot step rabbit_exchange_type_fanout defined by app rabbit
2024-09-13 17:57:12.591011+00:00 [info] <0.254.0> Running boot step rabbit_exchange_type_headers defined by app rabbit
2024-09-13 17:57:12.591109+00:00 [info] <0.254.0> Running boot step rabbit_exchange_type_topic defined by app rabbit
2024-09-13 17:57:12.591172+00:00 [info] <0.254.0> Running boot step rabbit_mirror_queue_mode_all defined by app rabbit
2024-09-13 17:57:12.591234+00:00 [info] <0.254.0> Running boot step rabbit_mirror_queue_mode_exactly defined by app rabbit
2024-09-13 17:57:12.591294+00:00 [info] <0.254.0> Running boot step rabbit_mirror_queue_mode_nodes defined by app rabbit
2024-09-13 17:57:12.591376+00:00 [info] <0.254.0> Running boot step rabbit_priority_queue defined by app rabbit
2024-09-13 17:57:12.591416+00:00 [info] <0.254.0> Priority queues enabled, real BQ is rabbit_variable_queue
2024-09-13 17:57:12.591519+00:00 [info] <0.254.0> Running boot step rabbit_queue_location_client_local defined by app rabbit
2024-09-13 17:57:12.591584+00:00 [info] <0.254.0> Running boot step rabbit_queue_location_min_masters defined by app rabbit
2024-09-13 17:57:12.591648+00:00 [info] <0.254.0> Running boot step rabbit_queue_location_random defined by app rabbit
2024-09-13 17:57:12.591705+00:00 [info] <0.254.0> Running boot step rabbit_exchange_type_consistent_hash_registry defined by app rabbitmq_consistent_hash_exchange
2024-09-13 17:57:12.591784+00:00 [info] <0.254.0> Running boot step kernel_ready defined by app rabbit
2024-09-13 17:57:12.591837+00:00 [info] <0.254.0> Running boot step ldap_pool defined by app rabbitmq_auth_backend_ldap
2024-09-13 17:57:12.591912+00:00 [info] <0.336.0> Starting worker pool 'ldap_pool' with 64 processes in it
2024-09-13 17:57:12.595597+00:00 [info] <0.254.0> Running boot step rabbit_sysmon_minder defined by app rabbit
2024-09-13 17:57:12.595835+00:00 [info] <0.254.0> Running boot step rabbit_epmd_monitor defined by app rabbit
2024-09-13 17:57:12.597011+00:00 [info] <0.505.0> epmd monitor knows us, inter-node communication (distribution) port: 25672
2024-09-13 17:57:12.597153+00:00 [info] <0.254.0> Running boot step guid_generator defined by app rabbit
2024-09-13 17:57:12.658032+00:00 [info] <0.254.0> Running boot step rabbit_node_monitor defined by app rabbit
2024-09-13 17:57:12.658376+00:00 [info] <0.509.0> Starting rabbit_node_monitor (in autoheal mode)
2024-09-13 17:57:12.658522+00:00 [info] <0.254.0> Running boot step delegate_sup defined by app rabbit
2024-09-13 17:57:12.659304+00:00 [info] <0.254.0> Running boot step rabbit_memory_monitor defined by app rabbit
2024-09-13 17:57:12.659514+00:00 [info] <0.254.0> Running boot step rabbit_fifo_dlx_sup defined by app rabbit
2024-09-13 17:57:12.659589+00:00 [info] <0.254.0> Running boot step core_initialized defined by app rabbit
2024-09-13 17:57:12.659621+00:00 [info] <0.254.0> Running boot step rabbit_channel_tracking_handler defined by app rabbit
2024-09-13 17:57:12.659705+00:00 [info] <0.254.0> Running boot step rabbit_connection_tracking_handler defined by app rabbit
2024-09-13 17:57:12.659754+00:00 [info] <0.254.0> Running boot step rabbit_definitions_hashing defined by app rabbit
2024-09-13 17:57:12.659858+00:00 [info] <0.254.0> Running boot step rabbit_exchange_parameters defined by app rabbit
2024-09-13 17:57:12.660131+00:00 [info] <0.254.0> Running boot step rabbit_mirror_queue_misc defined by app rabbit
2024-09-13 17:57:12.660612+00:00 [info] <0.254.0> Running boot step rabbit_policies defined by app rabbit
2024-09-13 17:57:12.661007+00:00 [info] <0.254.0> Running boot step rabbit_policy defined by app rabbit
2024-09-13 17:57:12.661070+00:00 [info] <0.254.0> Running boot step rabbit_queue_location_validator defined by app rabbit
2024-09-13 17:57:12.661136+00:00 [info] <0.254.0> Running boot step rabbit_quorum_memory_manager defined by app rabbit
2024-09-13 17:57:12.661191+00:00 [info] <0.254.0> Running boot step rabbit_quorum_queue defined by app rabbit
2024-09-13 17:57:12.661281+00:00 [info] <0.254.0> Running boot step rabbit_stream_coordinator defined by app rabbit
2024-09-13 17:57:12.661400+00:00 [info] <0.254.0> Running boot step rabbit_vhost_limit defined by app rabbit
2024-09-13 17:57:12.661457+00:00 [info] <0.254.0> Running boot step rabbit_federation_parameters defined by app rabbitmq_federation
2024-09-13 17:57:12.661579+00:00 [info] <0.254.0> Running boot step rabbit_federation_supervisor defined by app rabbitmq_federation
2024-09-13 17:57:12.669845+00:00 [info] <0.254.0> Running boot step rabbit_federation_queue defined by app rabbitmq_federation
2024-09-13 17:57:12.670027+00:00 [info] <0.254.0> Running boot step rabbit_federation_upstream_exchange defined by app rabbitmq_federation
2024-09-13 17:57:12.670120+00:00 [info] <0.254.0> Running boot step rabbit_mgmt_reset_handler defined by app rabbitmq_management
2024-09-13 17:57:12.670185+00:00 [info] <0.254.0> Running boot step rabbit_mgmt_db_handler defined by app rabbitmq_management_agent
2024-09-13 17:57:12.670224+00:00 [info] <0.254.0> Management plugin: using rates mode 'basic'
2024-09-13 17:57:12.670454+00:00 [info] <0.254.0> Running boot step rabbit_shovel_parameters defined by app rabbitmq_shovel
2024-09-13 17:57:12.670545+00:00 [info] <0.254.0> Running boot step recovery defined by app rabbit
2024-09-13 17:57:12.683991+00:00 [info] <0.559.0> Making sure data directory '/opt/bitnami/rabbitmq/.rabbitmq/mnesia/rabbit@mgp-tip-report-portal-rabbitmq-0.mgp-tip-report-portal-rabbitmq-headless.default.svc.cluster.local/msg_sto
res/vhosts/628WB79CIFDYO9LJI6DKMI09L' for vhost '/' exists
2024-09-13 17:57:12.769390+00:00 [info] <0.559.0> Starting message stores for vhost '/'
2024-09-13 17:57:12.769693+00:00 [info] <0.568.0> Message store "628WB79CIFDYO9LJI6DKMI09L/msg_store_transient": using rabbit_msg_store_ets_index to provide index
2024-09-13 17:57:12.773359+00:00 [info] <0.559.0> Started message store of type transient for vhost '/'
2024-09-13 17:57:12.773596+00:00 [info] <0.572.0> Message store "628WB79CIFDYO9LJI6DKMI09L/msg_store_persistent": using rabbit_msg_store_ets_index to provide index
2024-09-13 17:57:12.777476+00:00 [info] <0.559.0> Started message store of type persistent for vhost '/'
2024-09-13 17:57:12.777663+00:00 [info] <0.559.0> Recovering 0 queues of type rabbit_classic_queue took 92ms
2024-09-13 17:57:12.777726+00:00 [info] <0.559.0> Recovering 0 queues of type rabbit_quorum_queue took 0ms
2024-09-13 17:57:12.777782+00:00 [info] <0.559.0> Recovering 0 queues of type rabbit_stream_queue took 0ms
2024-09-13 17:57:12.779846+00:00 [info] <0.254.0> Running boot step empty_db_check defined by app rabbit
2024-09-13 17:57:12.779919+00:00 [info] <0.254.0> Will not seed default virtual host and user: have definitions to load...
2024-09-13 17:57:12.779990+00:00 [info] <0.254.0> Running boot step rabbit_observer_cli defined by app rabbit
2024-09-13 17:57:12.780111+00:00 [info] <0.254.0> Running boot step rabbit_looking_glass defined by app rabbit
2024-09-13 17:57:12.780155+00:00 [info] <0.254.0> Running boot step rabbit_core_metrics_gc defined by app rabbit
2024-09-13 17:57:12.780337+00:00 [info] <0.254.0> Running boot step background_gc defined by app rabbit
2024-09-13 17:57:12.780531+00:00 [info] <0.254.0> Running boot step routing_ready defined by app rabbit
2024-09-13 17:57:12.780588+00:00 [info] <0.254.0> Running boot step pre_flight defined by app rabbit
2024-09-13 17:57:12.780682+00:00 [info] <0.254.0> Running boot step notify_cluster defined by app rabbit
2024-09-13 17:57:12.780764+00:00 [info] <0.254.0> Running boot step networking defined by app rabbit
2024-09-13 17:57:12.780959+00:00 [info] <0.254.0> Running boot step rabbit_quorum_queue_periodic_membership_reconciliation defined by app rabbit
2024-09-13 17:57:12.781420+00:00 [info] <0.254.0> Running boot step definition_import_worker_pool defined by app rabbit
2024-09-13 17:57:12.781574+00:00 [info] <0.336.0> Starting worker pool 'definition_import_pool' with 16 processes in it
2024-09-13 17:57:12.782892+00:00 [info] <0.254.0> Running boot step cluster_name defined by app rabbit
2024-09-13 17:57:12.782990+00:00 [info] <0.254.0> Setting cluster name to 'mgp-tip-report-portal-rabbitmq' as configured
2024-09-13 17:57:12.786182+00:00 [info] <0.254.0> Running boot step virtual_host_reconciliation defined by app rabbit
2024-09-13 17:57:12.786429+00:00 [info] <0.254.0> Running boot step direct_client defined by app rabbit
2024-09-13 17:57:12.786588+00:00 [info] <0.254.0> Running boot step rabbit_federation_exchange defined by app rabbitmq_federation
2024-09-13 17:57:12.786857+00:00 [info] <0.254.0> Running boot step rabbit_management_load_definitions defined by app rabbitmq_management
2024-09-13 17:57:12.787027+00:00 [info] <0.623.0> Resetting node maintenance status
2024-09-13 17:57:13.274266+00:00 [warning] <0.651.0> Deprecated features: `management_metrics_collection`: Feature `management_metrics_collection` is deprecated.
2024-09-13 17:57:13.274266+00:00 [warning] <0.651.0> By default, this feature can still be used for now.
2024-09-13 17:57:13.274266+00:00 [warning] <0.651.0> Its use will not be permitted by default in a future minor RabbitMQ version and the feature will be removed from a future major RabbitMQ version; actual versions to be determine
d.
2024-09-13 17:57:13.274266+00:00 [warning] <0.651.0> To continue using this feature when it is not permitted by default, set the following parameter in your configuration:
2024-09-13 17:57:13.274266+00:00 [warning] <0.651.0>     "deprecated_features.permit.management_metrics_collection = true"
2024-09-13 17:57:13.274266+00:00 [warning] <0.651.0> To test RabbitMQ as if the feature was removed, set this in your configuration:
2024-09-13 17:57:13.274266+00:00 [warning] <0.651.0>     "deprecated_features.permit.management_metrics_collection = false"
2024-09-13 17:57:24.082888+00:00 [info] <0.688.0> Management plugin: HTTP (non-TLS) listener started on port 15672
2024-09-13 17:57:24.083134+00:00 [info] <0.718.0> Statistics database started.
2024-09-13 17:57:24.083286+00:00 [info] <0.717.0> Starting worker pool 'management_worker_pool' with 3 processes in it
2024-09-13 17:57:24.161500+00:00 [info] <0.746.0> Peer discovery: enabling node cleanup (will only log warnings). Check interval: 10 seconds.
2024-09-13 17:57:24.162218+00:00 [warning] <0.758.0> LDAP plugin loaded, but rabbit_auth_backend_ldap is not in the list of auth_backends. LDAP auth will not work.
2024-09-13 17:57:24.165578+00:00 [info] <0.767.0> Prometheus metrics: HTTP (non-TLS) listener started on port 9419
2024-09-13 17:57:24.165761+00:00 [info] <0.623.0> Ready to start client connection listeners
2024-09-13 17:57:24.168192+00:00 [info] <0.811.0> started TCP listener on [::]:5672
 completed with 11 plugins.
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0> Server startup complete; 11 plugins started.
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_prometheus
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_shovel_management
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_federation
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_auth_backend_ldap
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_peer_discovery_k8s
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_consistent_hash_exchange
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_peer_discovery_common
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_shovel
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_management
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_management_agent
2024-09-13 17:57:24.478792+00:00 [info] <0.623.0>  * rabbitmq_web_dispatch
2024-09-13 17:57:24.482176+00:00 [info] <0.9.0> Time to start RabbitMQ: 30797 ms
```

**MinIO**
```
←[38;5;6m ←[38;5;5m17:56:37.47 ←[0m←[38;5;2mINFO ←[0m ==>
←[38;5;6m ←[38;5;5m17:56:37.48 ←[0m←[38;5;2mINFO ←[0m ==> ←[1mWelcome to the Bitnami minio container←[0m
←[38;5;6m ←[38;5;5m17:56:37.49 ←[0m←[38;5;2mINFO ←[0m ==> Subscribe to project updates by watching ←[1mhttps://github.com/bitnami/containers←[0m
←[38;5;6m ←[38;5;5m17:56:37.50 ←[0m←[38;5;2mINFO ←[0m ==> Submit issues and feature requests at ←[1mhttps://github.com/bitnami/containers/issues←[0m
←[38;5;6m ←[38;5;5m17:56:37.57 ←[0m←[38;5;2mINFO ←[0m ==> Upgrade to Tanzu Application Catalog for production environments to access custom-configured and pre-packaged software components. Gain enhanced features, including Softwar
e Bill of Materials (SBOM), CVE scan result reports, and VEX documents. To learn more, visit ←[1mhttps://bitnami.com/enterprise←[0m
←[38;5;6m ←[38;5;5m17:56:37.58 ←[0m←[38;5;2mINFO ←[0m ==>
←[38;5;6m ←[38;5;5m17:56:37.58 ←[0m←[38;5;2mINFO ←[0m ==> ** Starting MinIO setup **
←[38;5;6mminio ←[38;5;5m17:56:37.87 ←[0m←[38;5;2mINFO ←[0m ==> Starting MinIO in background...
←[38;5;6mminio ←[38;5;5m17:56:43.17 ←[0m←[38;5;2mINFO ←[0m ==> Adding local Minio host to 'mc' configuration...
←[38;5;6mminio ←[38;5;5m17:56:43.79 ←[0m←[38;5;2mINFO ←[0m ==> Stopping MinIO...
←[38;5;6m ←[38;5;5m17:56:44.07 ←[0m←[38;5;2mINFO ←[0m ==> ** MinIO setup finished! **

←[38;5;6mminio ←[38;5;5m17:56:44.17 ←[0m←[38;5;2mINFO ←[0m ==> ** Starting MinIO **
MinIO Object Storage Server
Copyright: 2015-2024 MinIO, Inc.
License: GNU AGPLv3 - https://www.gnu.org/licenses/agpl-3.0.html
Version: DEVELOPMENT.2024-09-09T16-59-28Z (go1.22.7 linux/amd64)

API: http://localhost:9000
WebUI: http://100.65.98.249:9001 http://127.0.0.1:9001

Docs: https://docs.min.io
WARN: Detected Linux kernel version older than 4.0 release, there are some known potential performance problems with this kernel version. MinIO recommends a minimum of 4.x linux kernel version for best performance
```

**Comment by c0demonky** [comment]
@Pink-Bumblebee  im unable to upload files, so the index and ui logs cant be posted here. they're fairly long text files. but, we are able to access the ui and attempt to login.

<img width="960" alt="image" src="https://github.com/user-attachments/assets/dbb063e5-0fb2-4613-a2a8-bc6d9c54b885">

**Comment by hlebkanonik** [comment]
Can you please confirm the status of the migrations pod? Has it **Succeeded**?

**Comment by c0demonky** [comment]
What you see in the logs is all the information we have. here are those logs again
```
wait-for-it.sh: waiting for mgp-tip-report-portal-postgresql.default.svc.cluster.local:5432 without a timeout
wait-for-it.sh: waiting for opensearch-cluster-master.default.svc.cluster.local:9200 without a timeout
wait-for-it.sh: mgp-tip-report-portal-postgresql.default.svc.cluster.local:5432 is available after 15 seconds
no change
```
Pod is running

**Comment by hlebkanonik** [comment]
Can you connect to the database and check if something exists there? For example, public.schema_migrations can say how many migration scripts were applied. Looks like service migrations can't connect to the database to run migration scripts

Have you used our latest release, version 24.1.7?

**Comment by c0demonky** [comment]
how can i access the database? through shell only?

yes, using 24.1.7. i just redeployed it today. Commands:
```
git clone https://github.com/reportportal/kubernetes.git
helm dependency build reportportal
helm install mgp-tip-report-portal --set uat.superadminInitPasswd.password="MyPassword" ./reportportal
```

**Comment by Pink-Bumblebee** [comment]
@c0demonky , you could use external tools like DBeaver. Standard port is 5432. Username and password - from values.yaml or ask your admin.

**Comment by hlebkanonik** [comment]
Shell:
```bash
kubectl exec -it <DB POD NAME> -n <NAMESPACE> -- bash

> psql -U postgres -h localhost -d reportportal
> SELECT * FROM public.schema_migrations;
```

**Comment by c0demonky** [comment]
Output executing the above commands:
```
Password for user postgres:
psql (16.4)
Type "help" for help.

reportportal=# SELECT * FROM public.schema_migrations;
 version | dirty
---------+-------
      81 | f
(1 row)
```

**Comment by hlebkanonik** [comment]
Which version of Kubernetes are you using? It seems like Helm webhooks are not working properly. You can manually delete the migrations pod, all the services should run well.

**Comment by hlebkanonik** [comment]
Hi! Any updates?

**Comment by calebevans** [comment]
I was also running into this issue deploying 24.2 using the helm charts. I tried deleting the pod, and it just got re-created and the logs report "no change". I am using OpenShift 4.14 and the output of `SELECT * FROM public.schema_migrations;` was exactly the same as the reporter's output.

My quick hack to fix it was to delete the migration job itself, then had to go through all the deployments and delete the init containers and restart the deployments.

**Note:**
I also had to mount an empty directory to `/var/log/reportportal` in the API deployment. However, there is a good chance that had to do with OpenShift permissions rather than this specific issue.

**Comment by hlebkanonik** [comment]
Hi @calebevans, this might be an OpenShift issue because we tested it on a stock OpenShift cluster, and it worked fine except for the Routes. We’ll roll out a Routes update in the next release.

Have you used the [service account we provided](https://github.com/reportportal/kubernetes/blob/master/reportportal/templates/authorization/role.yaml#L12C1-L18C22) to work with the ReportPortal?

**Comment by hlebkanonik** [comment]
Closing due to no response from the requestor (label: Waiting Info From Requestor). If this is still relevant, please open a new issue with the requested information (e.g. environment details, logs, steps to reproduce) so we can help.
